src/WindowManager.vala

```diff
@@ -1391,6 +1408,8 @@ namespace Gala
 					(moving != null && window == moving))
 					continue;
 
+				kill_window_effects (actor);
```

Comment:
Do the same for all other windows that are not moving (but possibly maximizing / unmaximizing).